otrv4.md

```diff
@@ -644,10 +644,10 @@ roles), the expected attributes (expressed in fixed length) should be included.
 
 ### OTR Error Messages
 
-Any message containing the string "?OTR Error: " is an OTR Error Message. The
-following part of the message should contain human-readable details of the
-error. The message may also include a specific code at the beginning, e.g. "?OTR
-Error: ERROR_N: ". This code is used to identify which error is being
+An OTR Error message is the message that contains at the start position the string 
+"?OTR Error: ". The following part of the message should contain human-readable 
```

Comment:
Maybe: 

"Any message containing the string "?OTR Error: " at its start position is an OTR Error Message"

?

Reply:
I firstly thought to put in the way you wrote, after I thought that this looks strange at least for me, but if you think is better...